ledger/acctupdates.go

```diff
@@ -949,22 +965,19 @@ func (au *accountUpdates) commitRound(offset uint64, dbRound basics.Round, lookb
 
 	// create a copy of the deltas, round totals and protos for the range we're going to flush.
 	deltas := make([]map[basics.Address]accountDelta, offset, offset)
+	creatableDeltas := make([]map[basics.CreatableIndex]modifiedCreatable, offset, offset)
 	roundTotals := make([]AccountTotals, offset+1, offset+1)
 	protos := make([]config.ConsensusParams, offset+1, offset+1)
 	copy(deltas, au.deltas[:offset])
+	copy(creatableDeltas, au.creatableDeltas[:offset])
```

Comment:
Nit (not introduced by this PR): It would probably be good to add a comment explaining why deep copies of maps in `creatableDeltas` (and `deltas`) are not required here.